script/script.go

```diff
@@ -91,8 +91,7 @@ transaction(publicKey: String) {
 //
 // The returned balances include the value of the account's default FLOW vault,
 // as well as the optional FlowColdStorageProxy vault.
-const GetBalances = `import FlowColdStorageProxy from 0x{{.Contracts.FlowColdStorageProxy}}
-import FlowToken from 0x{{.Contracts.FlowToken}}
+const GetBalances = `import FlowToken from 0x{{.Contracts.FlowToken}}
```

Comment:
```suggestion
const GetBalances = `import FlowColdStorageProxy from 0x{{.Contracts.FlowColdStorageProxy}}
import FlowToken from 0x{{.Contracts.FlowToken}}
```

Reply:
I'll a dd a comment here to explain why we're not including ColdStorage contract